src/interpreter.rs

```diff
@@ -234,5 +247,7 @@ pub fn evaluate(program: &Program) -> Result<(String, String), Error> {
         }
     }
 
-    Ok((output, err_output))
+    let o = String::from_utf8(output).unwrap();
+    let e = String::from_utf8(err_output).unwrap();
```

Comment:
I'll add proper error types so we can write
```rust
Ok( String::from_utf8(output)?,  String::from_utf8(err_output)?)
```

What do you think?

Reply:
Yeah, would be nice.